server/schemas/item.graphql

```diff
@@ -27,12 +27,27 @@ type Item implements Node {
 
 union ItemEditor = User | Integration
 
-type ItemField {
+union ItemField = ItemValueField | ItemGroupField
+
+type ItemValueField {
   schemaFieldId: ID!
   type: SchemaFieldType!
   value: Any
 }
 
+type ItemGroupField {
+  id: ID!
+  groupId: ID!
+  schemaId: ID!
+  modelId: ID!
+  projectId: ID!
+  schema: Schema!
+  group: Group!
+  model: Model!
+  project: Project!
+  fields: [ItemField!]!
```

Comment:
```suggestion
  schemaFieldId: ID!
  groupId: ID!
  fields: [ItemField!]!
```

ItemValueField is simple, but this is too complex. Also this does not implements Node, so it can be simplified.